src/game_interpreter.h

```diff
@@ -398,4 +405,11 @@ inline AsyncOp Game_Interpreter::GetAsyncOp() const {
 	return _async_op;
 }
 
+#ifdef ENABLE_DYNAMIC_INTERPRETER_CONFIG
+inline void Game_Interpreter::ClearStateRuntimeFlags() {
+	_state.easyrpg_runtime_flags.conf_override_active = false;
+	_state.easyrpg_runtime_flags.flags.fill(false);
```

Comment:
clearing the flags shouldn't be necessary here (while we are discussing performance). They are only used when the override is true so you could them in the ``InterpreterFlags`` command when ``override_active == false`` instead.